daxa/mission/xmm.py

```diff
@@ -260,6 +260,11 @@ def _download_call(observation_id: str, insts: List[str], level: str, filename:
         :return: A None value.
         :rtype: Any
         """
+        # The astroquery module will only download to the cwd, so need to change to where we want the data downloaded
```

Comment:
Not going to require the changes, but I do not like comments on the end of lines personally - they can easily end up getting in the way/going over the maximum number of characters in a line when you change the code.